services/users.js

```diff
@@ -52,6 +52,20 @@ app.post('/services/users', passport.authenticate('signup'), function(req, res)
 	res.send(_.omit(req.user.toJSON(), "password"));
 });
 
+app.get( '/services/users', function ( req, res ) {
+	if ( req.isDev ) {
+		//new User({id:6}).destroy();
+		//new User({id:2}).destroy();
+		//new User({id:3}).destroy();
+		//new User({id:4}).destroy();
+		User.fetchAll().then( function ( list ) {
+			res.send( list );
+		});
+	} else {
+		res.status( 500 ).send( "Only available in development mode." );
+	}
+});
+
```

Comment:
It gives us a place to see the list of users and their properties while we're developing. I had to add it to check if the first user was an admin, and then to figure out why it wasn't. I wrapped it in the isDev thing afterwords which we won't be able to use as is until we have the 'donejs develop' thing added back with issue 12 which should be done earlyish this week.